src/main/java/org/portfolio/ourverse/src/persist/CustomCommentRepositoryImpl.java

```diff
@@ -0,0 +1,53 @@
+package org.portfolio.ourverse.src.persist;
+
+import com.querydsl.core.types.Order;
+import com.querydsl.core.types.OrderSpecifier;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import jakarta.persistence.EntityManager;
+import org.portfolio.ourverse.src.model.CommentOrderCondition;
+import org.portfolio.ourverse.src.persist.entity.Comment;
+import org.springframework.data.jpa.repository.support.QuerydslRepositorySupport;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.portfolio.ourverse.src.persist.entity.QComment.comment;
+
+@Repository
+public class CustomCommentRepositoryImpl extends QuerydslRepositorySupport implements CustomCommentRepository{
+
+    private final JPAQueryFactory jpaQueryFactory;
+
+    public CustomCommentRepositoryImpl(EntityManager entityManager) {
+        super(Comment.class);
+        this.jpaQueryFactory = new JPAQueryFactory(entityManager);
+    }
+
+    @Override
+    public List<Comment> findAllByFeed_IdWithCriteria(Long id, int pageNo, int pageSize, CommentOrderCondition form) {
+
+        return jpaQueryFactory.selectFrom(comment)
+                .where(comment.feed.id.eq(id))
+                .orderBy(getOrderSpecifiers(form))
+                .limit(pageSize)
+                .offset(pageNo)
+                .stream().toList();
+    }
+
+    private OrderSpecifier[] getOrderSpecifiers(CommentOrderCondition form) {
+        List<OrderSpecifier> orderSpecifiers = new ArrayList<>();
+
+        if(form.isCreatedAt()){
```

Comment:
```suggestion
        if (form.isCreatedAt()) {
```
if 문에 대한 convention 은 위와 같은데, 습관이 붙여서 하는걸로 되어있으신듯 합니다. 띄어쓰는걸로 꼭 맞추어주세요~!

https://naver.github.io/hackday-conventions-java/#space-around-brace